### DISCUSSION TITLE
🔔 Sunflower Land Info Telegram Channel 🌅

### DISCUSSION THREAD

**Discussion by WiseEyes** [sunflower-land/sunflower-land#2324] [General]
Hey farmers 👋 

I created a channel for Telegram users, where you can find out about current updates ✉

I also added emoji and sticker packs from the Sunflower Land 🌻

@sunflowerlandinfo 🔗

Hope you enjoy 🥰

**Comment by WiseEyes** [comment]
**Update**

**Added** emoji and stickers from the Easter event

**Comment by WiseEyes** [comment]
**Update**

**Corrected** the size of the emoji and Bears stickers. Added an Easter bear.

**Updated** Cabbage Boy, Cabbage Girl and Pablo the Bunny emoji and stickers.

**Added** emoji (exclamation point, question mark, check mark, heart) and some bumpkins and goblins.

**Comment by CarrotCounter** [comment]
Thanks @WiseEyes, I have joined your chat 👍

**Comment by WiseEyes** [comment]
**Update**

**Added** emoji and stickers from Dawn Breaker season.

**Updated** information about Dawn Breaker season.
